src/components/BigNumberPad.js

```diff
@@ -32,74 +32,68 @@ const padNumbers = [
     ['.', '0', '<'],
 ];
 
-class BigNumberPad extends React.PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            timer: null,
-        };
-    }
+const BigNumberPad = ({numberPressed, longPressHandlerStateChanged, nativeID, toLocaleDigit}) => {
```

Comment:
According to our [guidelines](https://github.com/Expensify/App/blob/08b1bf6af70507c7b65d1a1c6a4a5de7a29da6af/contributingGuides/STYLE.md#should-i-declare-my-components-with-arrow-functions-const-or-the-function-keyword), we prefer declaring these using the `function` keyword instead of `const`. We also try not to destructure props.

```suggestion
function BigNumberPad(props) {
```

Reply:
Gotcha!